packages/core/package.json

```diff
@@ -2,13 +2,13 @@
   "name": "@stryker-mutator/core",
   "version": "4.4.0",
   "description": "The extendable JavaScript mutation testing framework",
-  "main": "src/index.js",
+  "main": "dist/src/index.js",
```

Comment:
dist/src/index? so is it dist or src ;)

Reply:
Yes. So `src` is compiled to `dist/src`  and `test` is compiled to `dist/test`. If you really want `src` to be compiled to `dist` we would need to add the tests inside the `src` directory. I don't really mind this that much.

Reply:
I haven't noticed we have src and test in dist. If it is like this, it is ok i believe :)

Reply:
What would be consistent?